DashDoc.py

```diff
@@ -68,3 +68,16 @@ def run(self, edit, flip_syntax_sensitive=False, run_in_background=False):
         else:
             subprocess.call(['/usr/bin/open', '-g',
                          'dash-plugin://keys=%s&query=%s%s' % (','.join(keys), quote(query), background_string)])
+
+class DashDocSearchCommand(sublime_plugin.WindowCommand):
+    def run(self):
+        self.window.show_input_panel('Topic?', '', self.on_done, self.on_change, self.on_cancel)
+
+    def on_change(self, name):
+        pass
+
+    def on_cancel(self, name):
+        pass
+
+    def on_done(self, topic):
+        subprocess.call(["open", "dash://" + topic])
```

Comment:
I'd rather have this call to the [dash-plugin://](https://kapeli.com/dash_plugins) url too. note that you need to quote the query.

Reply:
Thank you @farcaller, will fix!